nitrogen8mm.coffee

```diff
@@ -0,0 +1,54 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+BOARD_POWEROFF = 'Remove and re-connect power to the board.'
+
+postProvisioningInstructions = [
+	instructions.BOARD_SHUTDOWN
+	instructions.REMOVE_INSTALL_MEDIA
+	instructions.BOARD_REPOWER
+]
+
+module.exports =
+	version: 1
+	slug: 'nitrogen8mm'
+	aliases: [ 'nitrogen8mm' ]
```

Comment:
This can be removed since this is a new board and does not need aliasing (aliasing is supposed to be used when we need to fix-up wrong slug names)